apps/builder/app/shared/nano-states/misc.ts

```diff
@@ -303,7 +303,17 @@ export const $hoveredInstanceSelector = atom<undefined | InstanceSelector>(
   undefined
 );
 
-export const $isPreviewMode = atom<boolean>(false);
+const builderModes = ["design", "preview", "edit"] as const;
```

Comment:
design -> build 😉

Reply:
decided to use "design" as in webflow build mode is our new "edit" mode

Reply:
gets a bit confusing, to me if its build/design is the same thing

Reply:
edit is very generic without any prefix like "content" ... I would actually call that "content" mode or "contentEditing"

Reply:
"edit" is semantically everything: design, build, content etc, all of it is edit